.circleci/config.yml

```diff
@@ -113,6 +116,20 @@ jobs:
             codecov --file ../karma_coverage_reports/lcov.info -F frontend
           when: on_success
 
+  frontend_test_coverage:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - run:
+          <<: *install_dependencies
+      - run: date +%F > date
+      - restore_cache:
+          <<: *restore_cache
```

Comment:
Please change it so it looks like https://github.com/oppia/oppia/blob/0737c09541af855409446792ae314ebd702da496/.circleci/config.yml#L77-L80.

Reply:
Done.